src/notation/internal/notationinteraction.cpp

```diff
@@ -1150,8 +1153,27 @@ void NotationInteraction::startDragCopy(const EngravingItem* element, QObject* d
         m_drag = nullptr;
     });
 
-    static QPixmap pixmap(2, 2); // null or 1x1 crashes on Linux under ChromeOS?!
+    const qreal adjustedRatio = 0.4;
+    const qreal width = element->ldata()->bbox().width();
+    const qreal height = element->ldata()->bbox().height();
+
+    QSize pixmapSize = QSize(width * adjustedRatio, height * adjustedRatio);
+    QPixmap pixmap(pixmapSize); // null or 1x1 crashes on Linux under ChromeOS?!
     pixmap.fill(Qt::white);
+    QBitmap mask = pixmap.createMaskFromColor(Qt::white); // Transparent background
+    pixmap.setMask(mask);
+
+    QPainter painter(&pixmap);
+    QPainter* qp = &painter;
+    const qreal dpi = qp->device()->logicalDpiX();
+
+    Painter p(qp, "startDragCopy");
```

Comment:
It looks like another slight implication is possible here:
```suggestion
    QPainter qp(&pixmap);
    const qreal dpi = qp.device()->logicalDpiX();

    Painter p(&qp, "startDragCopy");
```